src/tests/dispatch-payload.test.ts

```diff
@@ -151,11 +159,14 @@ Deno.test("DispatchPayload function file compatibility tests", async (t) => {
     },
   );
   Deno.chdir(origDir);
+  mockFetch.uninstall();
 });
 
 Deno.test("DispatchPayload with unhandled events", async (t) => {
+  mockFetch.install();
```

Comment:
We've been blasting the `complete*` APIs with bad requests every time we've ran the unit tests in this repo to date 😓

Reply:
Maybe we can take advantage of the testing [hooks: beforeAll, afterAll, beforeEach...](https://deno.com/manual@v1.33.3/basics/testing/behavior_driven_development#hooks) to prevent this from happening, but this might not be part of the scope of this PR

Reply:
@WilliamBergamin I added a new change to this PR to take a different approach to this problem: prevent the `deno test` command from having network access at all by dropping `--allow-net` from the task in `deno.jsonc`.